File: library/src/main/java/com/pengrad/telegrambot/response/UserChatBoostsResponse.java
package com.pengrad.telegrambot.response;

import com.pengrad.telegrambot.model.chatboost.ChatBoost;

import java.util.Arrays;

public class UserChatBoostsResponse extends BaseResponse {

    private ChatBoost[] boosts;

    public ChatBoost[] boosts() {
        return boosts;
    }

    @Override
    public String toString() {
        return "UserChatBoostsResponse{" +
            "boosts=" + Arrays.toString(boosts) +
            '}';
    }
}

File: library/src/main/java/com/pengrad/telegrambot/response/UserChatBoostsResponse.kt
package com.pengrad.telegrambot.response

import com.pengrad.telegrambot.model.chatboost.ChatBoost

data class UserChatBoostsResponse(private val result: ChatBoosts = ChatBoosts(emptyList())) : BaseResponse() {
    data class ChatBoosts(val boosts: List<ChatBoost>)

    fun boosts(): Array<ChatBoost> {
        return result.boosts.toTypedArray()
    }
}